docs/core/project-sdk/msbuild-props.md

```diff
@@ -73,16 +73,40 @@ You can specify properties such as `PackageId`, `PackageVersion`, `PackageIcon`,
 </PropertyGroup>
 ```
 
-## Publish properties and items
+## Publish properties, items, and metadata
 
 - [AppendRuntimeIdentifierToOutputPath](#appendruntimeidentifiertooutputpath)
 - [AppendTargetFrameworkToOutputPath](#appendtargetframeworktooutputpath)
 - [CopyLocalLockFileAssemblies](#copylocallockfileassemblies)
+- [CopyToPublishDirectory](#copytopublishdirectory)
+- [LinkBase](#linkbase)
 - [RuntimeIdentifier](#runtimeidentifier)
 - [RuntimeIdentifiers](#runtimeidentifiers)
 - [TrimmerRootAssembly](#trimmerrootassembly)
 - [UseAppHost](#useapphost)
 
+### CopyToPublishDirectory
+
+The `CopyToPublishDirectory` metadata on an MSBuild item controls when the item is copied to the publish directory. Allowable values are `PreserveNewest`, which only copies the item if it has changed, and `Always`, which always copies the item. From a performance standpoint, `PreserveNewest` is preferable because it enables an incremental build.
+
+```xml
+<ItemGroup>
+  <None Update="**/my_pattern/**" CopyToOutputDirectory="PreserveNewest" CopyToPublishDirectory="PreserveNewest" />
+</ItemGroup>
+```
+
+### LinkBase
+
+For an item that's outside of the project directory and its subdirectories (the *project cone*), the publish target uses the item's [Link metadata](/visualstudio/msbuild/common-msbuild-item-metadata) to determine where to copy the item to. `Link` also determines how items outside of the project cone are displayed in the Solution Explorer window of Visual Studio.
```

Comment:
This is the first time I've heard the term "project cone", is it used a lot? I've always referred to this as the "project root", but I do like "cone" - it helps to visualize it a bit.

Reply:
Also, it's not used again in this doc, so maybe it doesn't need to be introduced at all.  The common-msbuild-item-metadata doc that is linked to mentions the "project tree".

Reply:
> `Link` also determines how items outside of the project cone are displayed in the **Solution Explorer** window of Visual Studio.

It might be helpful to describe or show what that actually looks like in Solution Explorer.

Reply:
I saw cone used [here](https://github.com/dotnet/sdk/issues/1115) and liked it, but I agree, better not to introduce new terminology.